code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -0,0 +1,252 @@
+/datum/antagonist/heretic
+	name = "Heretic"
+	roundend_category = "Heretics"
+	antagpanel_category = "Heretic"
+	antag_moodlet = /datum/mood_event/heretics
+	job_rank = ROLE_HERETIC
+	var/antag_hud_type = ANTAG_HUD_HERETIC
+	var/antag_hud_name = "heretic"
+	var/give_equipment = TRUE
+	var/list/researched_knowledge = list()
+	var/total_sacrifices = 0
+
+/datum/antagonist/heretic/admin_add(datum/mind/new_owner,mob/admin)
+	give_equipment = FALSE
+	new_owner.add_antag_datum(src)
+	message_admins("[key_name_admin(admin)] has heresized [key_name_admin(new_owner)].")
+	log_admin("[key_name(admin)] has heresized [key_name(new_owner)].")
+
+/datum/antagonist/heretic/greet()
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ecult_op.ogg', 100, FALSE, pressure_affected = FALSE)//subject to change
+	to_chat(owner, "<span class='boldannounce'>You are the Heretic!</span><br>\
+	<B>The old ones gave you these tasks to fulfill:</B>")
+	owner.announce_objectives()
+	to_chat(owner, "<span class='cult'>The book whispers, the forbidden knowledge walks once again!<br>\
+	Your book allows you to research abilities, read it very carefully! you cannot undo what has been done!<br>\
+	You gain charges by either collecitng influences or sacrifcing people tracked by the living heart<br> \
+	You can find a basic guide at : https://tgstation13.org/wiki/Heresy_101 </span>")
+
+/datum/antagonist/heretic/on_gain()
+	var/mob/living/current = owner.current
+	if(ishuman(current))
+		forge_primary_objectives()
+		gain_knowledge(/datum/eldritch_knowledge/spell/basic)
+		gain_knowledge(/datum/eldritch_knowledge/living_heart)
+		gain_knowledge(/datum/eldritch_knowledge/codex_cicatrix)
+	current.log_message("has been converted to the cult of the forgotten ones!", LOG_ATTACK, color="#960000")
```

Comment:
Having this as 'has been converted to a heretic' would probably be better for admins